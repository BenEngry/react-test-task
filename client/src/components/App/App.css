body {
    margin: 0px;
    padding: 0px;

    width: 100%;
    background-color:rgb(48, 18, 105);

    color: #fff;
    font-family: 'Courier New', Courier, monospace;
    font-size: 25px;
}

.wrapper {
    margin: 250px auto 0px;
    width: 60%;
    padding: 20px 40px;
    background-color: rgb(106, 62, 189);
    border-radius: 20px;
}