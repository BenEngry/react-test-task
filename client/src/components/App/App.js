import React from "react";
import { BrowserRouter } from "react-router-dom";
import AppRouter from "../AppRouter/AppRouter";
import "./App.css";

export const App = () => {
	return (
		<BrowserRouter>
            <AppRouter />
    	</BrowserRouter>
	);
}

export default App;