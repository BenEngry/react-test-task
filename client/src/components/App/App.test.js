import renderer from "react-test-renderer";
import React from "react";
import { App } from "./App";

describe("Testing <App/>", ()=> {

    it("User have rendered correctly", ()=>{
        const app = renderer.create(<App/>).toJSON();
        expect(app).toMatchSnapshot();
    });

});
