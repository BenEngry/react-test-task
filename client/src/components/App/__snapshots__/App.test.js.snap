// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <App/> User have rendered correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="loader"
  >
    Loading...
  </div>
</div>
`;
